### PR TITLE
Fix `MANIFEST.in` and make CI tests build and use a source distribution instead of the source directly

### PR DESCRIPTION
**Pull request by gusmonod**
#44 was closed when I (accidentally) force pushed to the branch, removing all commits that weren't on `main`… Re-opening as another PR because I don't have the privileges required to re-open #44.

### CODE REVIEW

File: .github/workflows/build.yml
Comment (godlygeek) [inline .github/workflows/build.yml:7]: Let's not do this. It doesn't play well with required status checks (this will stop the workflow from running if the only file changed is CONTRIBUTING.md, which sounds like it saves work since we know the tests can't possibly fail in that case, but the PR can't be landed because the required status checks will never pass)

File: .github/workflows/build.yml
Comment (godlygeek) [inline .github/workflows/build.yml:90]: Ooh, runtime dependencies of comdb2, I guess? I didn't think about those.

File: .github/workflows/build.yml
Comment (godlygeek) [inline .github/workflows/build.yml:93]: What's the rationale for removing these? It doesn't seem harmful, and it's probably fast - but it doesn't seem helpful either, AFAICS. Is there a reason it's useful that I'm not seeing?

File: .github/workflows/build.yml
Comment (godlygeek) [inline .github/workflows/build.yml:97]: Whoa, it does!? Why?

File: .github/workflows/build.yml
Comment (godlygeek) [inline .github/workflows/build.yml:103]: If `actions/setup-python` is setting `PKG_CONFIG_PATH`, we should probably tack on our path to theirs:
```suggestion
          PKG_CONFIG_PATH=/opt/bb/lib/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig:$PKG_CONFIG_PATH
```
I'm assuming `/usr/lib/x86_64-linux-gnu/pkgconfig` is what they're setting it to?

File: .github/workflows/build.yml
Comment (godlygeek) [inline .github/workflows/build.yml:148]: Huh, it's odd that this wasn't required before and now is

File: .github/workflows/build.yml
Comment (godlygeek) [inline .github/workflows/build.yml:193]: Same comment as above

**Review by godlygeek — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml
Comment (gusmonod) [reply .github/workflows/build.yml:90]: Yep!

File: .github/workflows/build.yml
Comment (gusmonod) [reply .github/workflows/build.yml:93]: No real reason, just that we don't need them. I can remove the `rm` command as this will be less code to maintain.

File: .github/workflows/build.yml
Comment (gusmonod) [reply .github/workflows/build.yml:97]: 🤷🏼

File: .github/workflows/build.yml
Comment (gusmonod) [reply .github/workflows/build.yml:103]: Probably. Let's add `$PKG_CONFIG_PATH` at the end yes.

File: .github/workflows/build.yml
Comment (gusmonod) [reply .github/workflows/build.yml:148]: Hmm… I'm not sure why this was anymore, probably because it is added by `sudo make install`, but I tried to make this PR work with a .tar.gz archive of `/opt/bb`… This may not be needed, I'll try to remove it.

File: .github/workflows/build.yml
Comment (godlygeek) [reply .github/workflows/build.yml:93]: I don't think it _really_ matters either way - I was just wondering if there was a reason why it _must_ be done that I wasn't seeing. If not, your call on whether to keep it or remove it.

**Review by godlygeek — DISMISSED** [review]
Approving so that you can land this without another round of reviews, but I'd like the `PKG_CONFIG_PATH` and `paths-ignore` changes addressed before this lands.

File: .github/workflows/build.yml
Comment (godlygeek) [reply .github/workflows/build.yml:148]: Having it is fine if we need it, but I can't immediately spot a reason why it ought to be required.

**Comment by gusmonod** [comment]
> Approving so that you can land this without another round of reviews, but I'd like the `PKG_CONFIG_PATH` and `paths-ignore` changes addressed before this lands.

I guess that the repo config made it so that the review was dismissed as soon as I pushed something else anyway 😞 Oh well, I'll get someone else to approve this on Monday probably.

**Comment by gusmonod** [comment]
Changes since Matt approved this: https://github.com/bloomberg/python-comdb2/compare/6e4d75bc9b42b298f0243de7167ec65d9a28aecc..1bc9ad995007e95fbd8f8dddfaf58e9e914db0a8

**Review by godlygeek — APPROVED** [review]
LGTM

**Pull request merged**
